src/target/spi.c

```diff
@@ -121,8 +121,10 @@ spi_flash_s *bmp_spi_add_flash(target_s *const target, const target_addr_t begin
 		spi_parameters.sector_size = 4096U;
 		spi_parameters.capacity = length;
 		spi_parameters.sector_erase_opcode = SPI_FLASH_OPCODE_SECTOR_ERASE;
+		DEBUG_WARN("SFDP read failed. Using best guess.\n");
 	}
 	DEBUG_INFO("Flash size: %" PRIu32 "MiB\n", (uint32_t)spi_parameters.capacity / (1024U * 1024U));
+	DEBUG_INFO("Flash page size: %" PRIu32 "B\n", (uint32_t)spi_parameters.page_size);
```

Comment:
We can see why you'd want to print this parameter out, but we don't think it adds to the information displayed by this routine for the Flash cost associated with it. The warning above is good though to help indicate control flow to the user.